src/scss/_typography.scss

```diff
@@ -151,36 +180,44 @@ h5,
 }
 
 .text-body-lg {
-  @apply text-lg;
-  line-height: 1.875rem; // 30px
+  font-size: pxToRem(18);
```

Comment:
Could be overkill, and feel free to disregard if you don't agree, but I'd suggest noting which Tailwind `fontSize` these correspond to. Purposes being that there is some rhyme/reason as to where these sizes came from for `text-body-*`, and if we ever make changes, we may want to consider changing the text scale elsewhere.